src/main/java/graphql/annotations/GraphQLAnnotations.java

```diff
@@ -511,7 +511,10 @@ private GraphQLOutputType getGraphQLConnection(boolean isConnection, AccessibleO
     private boolean isConnection(AccessibleObject obj, Class<?> klass, GraphQLOutputType type) {
         return obj.isAnnotationPresent(GraphQLConnection.class) &&
                 type instanceof GraphQLList &&
-                ((GraphQLList) type).getWrappedType() instanceof GraphQLObjectType;
+                (((GraphQLList) type).getWrappedType() instanceof GraphQLObjectType ||
```

Comment:
Maybe its better to initialize a set of valid types (Set <Class<?>> validTypes) and then ask if (GraphQLList) type).getWrappedType().class is in the set. (Or other type of set, maybe strings of names?)

Reply:
It could be better in terms of performances, but as the classes are not final and new sub-classes may eventually appear in graphql-java, I think it's better to keep "instanceof" tests instead of just checking if the class itself is in a list of predefined class, that will be more hard to maintain.
But instead of simple contains, we could also have the list of types in a collection and check if the type is an instance of any of them by using a lambda, could be a little nicer.
What do you think ?

Reply:
Sounds good!

Reply:
ok, done